maven-plugin-tools-generators/src/main/java/org/apache/maven/tools/plugin/generator/PluginXdocGenerator.java

```diff
@@ -405,8 +395,7 @@ private void writeGoalAttributes( MojoDescriptor mojoDescriptor, XMLWriter w )
      */
     private void writeGoalParameterTable( MojoDescriptor mojoDescriptor, XMLWriter w )
     {
-        @SuppressWarnings( "unchecked" )
-        List<Parameter> parameterList = mojoDescriptor.getParameters();
+        @SuppressWarnings( "unchecked" ) List<Parameter> parameterList = mojoDescriptor.getParameters();
```

Comment:
Is the suppressWarning still required?

Reply:
I guess not but I didn't upgrade to maven 3 :)